scipy/special/_basic.py

```diff
@@ -2176,6 +2177,18 @@ def comb(N, k, exact=False, repetition=False):
     repetition : bool, optional
```

Comment:
This is also another limitation compared to factorial, which switches to python integers instead of overflowing.

Reply:
> This is also another limitation compared to factorial, which switches to python integers instead of overflowing.

`comb` also switches to python integers instead of overflowing. I've pasted the relevant code below. It can be found [here](https://github.com/scipy/scipy/blob/c4bd4ac28561a6f9620240ded15909401ccf3690/scipy/special/_comb.pyx#L5-L30).

```python
def _comb_int(N, k):
    # Fast path with machine integers
    try:
        r = _comb_int_long(N, k)
        if r != 0:
            return r
    except (OverflowError, TypeError):
        pass


    # Fallback
    N = int(N)
    k = int(k)


    if k > N or N < 0 or k < 0:
        return 0


    M = N + 1
    nterms = min(k, N - k)


    numerator = 1
    denominator = 1
    for j in range(1, nterms + 1):
        numerator *= M - j
        denominator *= j


    return numerator // denominator
```

Reply:
Fair point, I read over this too quickly. Thanks for double-checking.